LibShifterBox/LibShifterBox.lua

```diff
@@ -322,6 +393,10 @@ local function _removeEntriesFromList(list, keys)
         if removedKey ~= nil then hasAtLeastOneRemoved = true end
     end
     if hasAtLeastOneRemoved then
+        -- then trigger the callback if present
+        _fireCallback(list.shifterBox, list, { [true] = lib.EVENT_LEFT_LIST_ENTRY_REMOVED, [false] = lib.EVENT_RIGHT_LIST_ENTRY_REMOVED  }, list.shifterBox.isLeftList,
```

Comment:
I do not seem to understand you?

It controls the different event handlers used inside the function _fireCallback!
You pass in the table with the left (true) and right values (false).
Somehow the function needs the different event values (e.g. EVENT_LEFT_LIST_ENTRY_REMOVED and EVENT_RIGHT_LIST_ENTRY_REMOVED), so the table was the easiest way to achieve this.

Reply:
I was rather thinking of something like this:
`_fireCallback(list.shifterBox, list, (list.shifterBox.isLeftList and lib.EVENT_LEFT_LIST_ENTRY_REMOVED or lib.EVENT_RIGHT_LIST_ENTRY_REMOVED), list.shifterBox.isLeftList,`
This way you can remove the check inside the "_fireCallback" function to figure out which event should be fired, because the function will only ever receive one event as a parameter. Makes it also more easy to read/understand from outside the function as you know exactly which event is passed on to the function, whereas in the current implementaiton this is not clear.

Reply:
Good idea, implemented it.